multiqc/utils/search_patterns.yaml

```diff
@@ -60,6 +60,8 @@ htseq:
     contents: '__too_low_aQual'
 hicup:
     fn: 'HiCUP_summary_report*'
+jellyfish:
+    fn: '*.hist'
```

Comment:
Ah ok, then yes just describe the behaviour in the docs then. No need for a new setup I think, can just use existing config options: http://multiqc.info/docs/#module-search-patterns

Config can now also be set on the [command line](http://multiqc.info/docs/#command-line-config) so can already do the that if people don't want to mess with config files. Something like this (untested):
```
multiqc --cl_config "sp: { jellyfish: { fn: '*.whatever' } }"
```